docs/src/pages/components/dialogs/CustomizedDialogs.js

```diff
@@ -1,59 +1,44 @@
 import * as React from 'react';
-import { withStyles } from '@material-ui/core/styles';
+import PropTypes from 'prop-types';
 import Button from '@material-ui/core/Button';
 import Dialog from '@material-ui/core/Dialog';
 import MuiDialogTitle from '@material-ui/core/DialogTitle';
```

Comment:
```suggestion
import DialogTitle from '@material-ui/core/DialogTitle';
```

Should we rename this import too?

Reply:
It was renamed because a custom component was created here 

https://github.com/mui-org/material-ui/blob/8264f146f7e34f2cf62b1db9feb61ff6f17f1fdc/docs/src/pages/components/dialogs/CustomizedDialogs.js#L12

Maybe the custom component should be called something else?

Reply:
Ah got ya, let's leave it as is then :)

Reply:
I have personally prefixed components with MuiX in the past when I have faced this case (outside of this codebase). In this codebase, it seems that we have tried to give descriptive names to the customizations. Here, it's a lot like Bootstrap.

Reply:
+1, I used `BootstrapTooltip` as a name here https://github.com/mui-org/material-ui/pull/26165/files#diff-4240400700836766880919f7642c12d598eba8fdbfa20552cf05a1af817c21d5R18 Let's do that then

Reply:
Seem right. `BootstrapDialogTitle` or `BsDialogTitle`?

Reply:
`BootstrapDialogTitle`